extmod/vfs_fat.c

```diff
@@ -99,7 +99,7 @@ STATIC mp_obj_t fat_vfs_make_new(const mp_obj_type_t *type, size_t n_args, size_
 }
 
 STATIC void verify_fs_writable(fs_user_mount_t *vfs) {
```

Comment:
Since this is no longer just a check, I'd call this something like `filesystem_lock_raise()` instead.